lib/webpack-loader.js

```diff
@@ -1,10 +1,14 @@
 import {getOptions} from 'loader-utils'
 import {compile} from './core.js'
 
-// A Webpack (4+) loader for xdm.
-// See `webpack.cjs`, which wraps this, because Webpack loaders must currently
-// be CommonJS.
-// `file.map` is defined when a `SourceMapGenerator` is passed in options.
+/**
+ * A Webpack (4+) loader for xdm.
+ * See `webpack.cjs`, which wraps this, because Webpack loaders must currently
+ * be CommonJS.
+ * `file.map` is defined when a `SourceMapGenerator` is passed in options.
+ * @param {string} value
+ * @param {*} callback
```

Comment:
haven't found the exact callback construct in the webpack docs or typings yet

Reply:
`callback` comes from `this.async()` https://github.com/wooorm/xdm/blob/21c18aeea2e0f29cdbac7488ef54745ff8f749e8/webpack.cjs#L5

Reply:
thanks! updated